dashboard/app/controllers/code_reviews_controller.rb

```diff
@@ -0,0 +1,62 @@
+class CodeReviewsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Do we want this on `index`?

Reply:
Yes, I think we'll still need `current_user` for `index`?

Reply:
I was forgetting the purpose of `authenticate_user`, it will set the `current_user`, got it. Yeah this makes sense